docs/shared/tpl/stats.html

```diff
@@ -12,7 +12,7 @@
         platforms => 100,
 
         # Years since December 18, 1987
-        perl_age => 29,
+        perl_age => 30,
```

Comment:
Could these numbers be auto-calculated with template toolkit?

Reply:
I don't know if they are, I'm not knowledgeable with this stack. I'm thinking it may not be automatic because I changed it almost 3 days after it should have?

Reply:
@harpo109 Yes, it's not. :-) I wasn't clear, I think -- 

I was suggesting instead of just changing it then use Plugin::Date to calculate the number of years. http://search.cpan.org/dist/Template-Toolkit/lib/Template/Plugin/Date.pm (or some variation of that).